egs2/librispeech/asr1/README.md

```diff
@@ -1,3 +1,37 @@
+# RNN-T Conformer
+
+## Environments
+- date: `Fri Mar 25 04:35:42 EDT 2022`
+- python version: `3.9.5 (default, Jun  4 2021, 12:28:51)  [GCC 7.5.0]`
+- espnet version: `espnet 0.10.7a1`
+- pytorch version: `pytorch 1.8.1+cu111`
+- Git hash: `21d19be00089678ca27f7fce474ef8d787689512`
+  - Commit date: `Wed Mar 16 08:06:52 2022 -0400`
+- ASR config: [conf/train_rnnt_conformer.yaml](conf/train_rnnt_conformer.yaml)
```

Comment:
```suggestion
- ASR config: [conf/tuning/transducer/train_conformer-rnn_transducer.yaml](conf/train_rnnt_conformer.yaml)
```

Reply:
Thank you for the great suggestions. Also, I will try to train the model from #4032 when my cluster frees up.